docs/source/couchdb_as_state_database.rst

```diff
@@ -223,6 +229,13 @@ Docker compose scripts only set the username and password at the creation of
 the container. The *local.ini* file must be edited if the username or password
 is to be changed after creation of the container.
 
+If you choose map the fabric-couchdb container port to a host port, make sure you
+are aware of the security implications. Mapping the CouchDB container port in a
+development environment exposes the CouchDB REST API and allows you to visualize
+the database via the CouchDB web interface (Fauxton). In a production environment
+you should refrain from mapping the host port to restrict access to the CouchDB
```

Comment:
"restrict external client access to the CouchDB container"

...the peer container will still have access of course.

Reply:
Added: Only the peer will be able to access the CouchDB container.